lib/IRGen/GenClangType.cpp

```diff
@@ -475,8 +475,8 @@ GenClangType::visitBoundGenericType(CanBoundGenericType type) {
   auto args = type.getGenericArgs();
   assert(args.size() == 1 &&
          "should have a single generic argument!");
-  auto loweredArgTy = IGM.getLoweredType(args[0]).getASTType();
-
+  auto argSILType = IGM.getLoweredType(args[0]);
+  auto argLoweredTy = argSILType.getASTType();
```

Comment:
I don't understand why you've changed this variable name; it seems spurious.

Reply:
This change was made before the update to the getASTType documentation. Forgot to remove it.